develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -0,0 +1,381 @@
+---
+title: Logs
+sidebar_order: 3
+---
+
+<Alert title="Note" level="warning">
+
+The Sentry Logs feature is under active development. The information in this document is subject to change.
+
+</Alert>
+
+This document defines the format used by Sentry to ingest logs, as well as the SDK API and behavior that create and send logs to Sentry.
+
+## Logs Protocol
+
+There are two ways to send logs to Sentry: via the `log` envelope and Sentry Log protocol (preferred), or the `otel_log` envelope and OpenTelemetry Log protocol.
+
+### `otel_log` Envelope Item Payload
+
+The `otel_log` envelope item payload is a JSON object that represents an OpenTelemetry Log. Multiple `otel_log` envelope items can be sent in a single envelope.
```

Comment:
We should mention, that this is an implemention of the OptenTelemetry Logs Data Model and link here: https://opentelemetry.io/docs/specs/otel/logs/data-model/